src/main/java/com/glodblock/github/nei/NEI_FC_Config.java

```diff
@@ -14,26 +14,36 @@ public void loadConfig() {
         API.registerNEIGuiHandler(new NEIGuiHandler());
         API.addSearchProvider(new NEIItemFilter());
         for (String identifier : FluidRecipe.getSupportRecipes()) {
-            API.registerGuiOverlayHandler(
-                    GuiFluidCraftingWireless.class,
-                    FluidCraftingTransferHandler.INSTANCE,
-                    identifier);
-            API.registerGuiOverlayHandler(
-                    GuiFluidPatternWireless.class,
-                    FluidPatternTerminalRecipeTransferHandler.INSTANCE,
-                    identifier);
-            API.registerGuiOverlayHandler(
-                    GuiFluidPatternExWireless.class,
-                    FluidPatternTerminalRecipeTransferHandler.INSTANCE,
-                    identifier);
-            API.registerGuiOverlayHandler(
-                    GuiFluidPatternTerminal.class,
-                    FluidPatternTerminalRecipeTransferHandler.INSTANCE,
-                    identifier);
-            API.registerGuiOverlayHandler(
-                    GuiFluidPatternTerminalEx.class,
-                    FluidPatternTerminalRecipeTransferHandler.INSTANCE,
-                    identifier);
+            if (!API.hasGuiOverlayHandler(GuiFluidCraftingWireless.class, identifier)) {
```

Comment:
Add a comment explaining that NEE handlers take priority

Reply:
![image](https://user-images.githubusercontent.com/33802162/218980074-caa750ae-6340-4f35-a894-a6d4295a8b75.png)